contracts/src/Teleporter/upgrades/tests/TeleporterRegistryTests.t.sol

```diff
@@ -252,7 +230,11 @@ contract TeleporterRegistryTest is Test {
 
         // Check if we have an invalid destination address
         warpMessage = _createWarpOffChainMessage(
-            latestVersion + 1, address(this), teleporterAddress, address(teleporterRegistry)
```

Comment:
Can you help me understand why `destinationAddress` was `address(this)` previously, and why this is no longer needed?

Reply:
it was set as `address(this)` to mock an invalid Warp message to the registry, we still set it in the next line

Reply:
Thanks!